### DISCUSSION TITLE
Works great! T128 on SteamOS with DKMS

### DISCUSSION THREAD

**Discussion by iliana** [Kimplul/hid-tmff2#120] [General]
Hey, wanted to reach out and say thanks for this! I wanted to post some "it works in this environment" notes to help maintainers out.

I've integrated this module (via DKMS) into [my SteamOS image repacking setup](https://git.iliana.fyi/fauxlo/commit/?id=6f615d0047acfa2e8087ddce8e06c142200ea657) and it works out-of-the-box with American Truck Simulator. This is particularly exciting because I bought a used T128 before even looking up the Linux support situation, so I'm quite happy :)

Notes:
- Thrustmaster T128
- SteamOS 3.5.19 on non-Deck hardware
- hid-tmff2 @ 542f3f002b4e1c3a8a5506ed9ec546fe28a890bf, using DKMS installation
- ATS 1.51.2.3
- ATS apparently has a native Linux build; the wheel has a really terrible deadzone in the native build (probably about 30-45° either direction from center), but forcing it to use the Windows build with Proton Experimental removes the deadzone entirely
- Force feedback works great running under Proton Experimental; did not need to install the TM drivers in the Wine prefix. (It looks like the game ships its own drivers for TM wheels?)
- If you are building patched kernels for SteamOS, don't build them on a newer Arch toolchain or you'll be missing compiler symbols when you want to add out-of-tree modules! Learned this the hard way...

**Comment by Kimplul** [comment]
Cheers! Always nice to see things working, I mostly get informed about things that are broken in some way.

There have been some reports that some wheels don't seem to send out input under Steam, #118 at least is also running SteamOS but on a deck. Consider this a warning, I suppose :) Of course, any and all help figuring it out is greatly appreciated.
